### ISSUE TITLE
Authenticated svn repo causes issues with ranger

### ISSUE THREAD

**Issue by SilverFishCat** [ranger/ranger#1581]
#### Runtime Environment

- Operating system and version: 5.1.5-arch1-2-ARCH
- Terminal emulator and version: alacritty 0.3.2
- Python version: 3.7.3 (default, Mar 26 2019, 21:43:19) [GCC 8.2.1 20181127]
- Ranger version/commit: ranger 1.9.2
- Locale: en_US.UTF-8

#### Current Behavior
While `vcs_backend_svn` is `enabled` or `local`, browsing to a directory that contains an svn directory that requires authentication to connect to the repo will prompt the svn login prompt alongside ranger. This will show the svn login prompt inline into the console somewhere(sometimes nowhere), and keyboard input will intermittenly be sent to the login prompt.
To the user, it looks like there a couple of section of ranger are parts of the log in prompt (though for me, ranger overriden it so i usually didn't even notice it had anything to do with svn) and a bunch of my inputs would generate `*` at the cursor, input that ranger would ignore until one of the inputs would actually reach ranger (happens both in normal mode and command mode, the asterisks would show up at the currently selected file and the command line, respectively).
Pressing ^C exits the prompt. Navigating to the directory again pops it up again.

#### Expected Behavior
Ranger should switch to the prompt exclusively or ignore svn repos that require authentication (or any other solution that does no create this unexpected behavior, as it was not easily apparent it was caused by svn).
Additionally, if `vcs_backend_svn` is set to `local`, there should be any communication with the svn repo. This might be more difficult as svn directories are not the same as local git repos and I don't know how much local information is actually saved and if the tools support offline operations that well (or that easily).

#### Context

I can't browse certain directories while having that option on without being bothered every time I look at them. Just having to work with directories adjacent to svn directories is annoying as there isn't a very good feedback to when the prompt is actually closed.

#### Possible Solutions
Find a way to present the prompt gracefully.
Alternatively, use the `--non-interactive` option so it doesn't actually print the prompt to screen, using cached tokens (cached tokens are an svn feature i think, not implying ranger should handle those).
Regardless, prevent the command from printing to the screen and disable the `local` value for `vcs_backend_svn` if it's not supported/not possible.

#### Steps to reproduce
1. Set `vcs_backend_svn` to `enabled`/`local`.
2. Create an svn working copy.
3. Wait for the cached tokens to expire.
4. Browse to that directory / browse to the parent directory and select the svn directory.

**Comment by toonn** [comment]
Good catch. Interactive prompts are challenging to handle. Unless someone steps up we'll probably go with `--non-interactive` or something else that just prevents prompts.
